### PR TITLE
💡Developers may see a full example of the normalized event data structure

### PR DESCRIPTION
**Pull request by zspencer**
OK, I didn't get to the architecture docs yet; but I  documented the core data
structure and the basic layout.

This helps with #44, and maybe would solve it completely if instead of a an `ARCHITECTURE.md` file we had an architecture section?

### CODE REVIEW

**Review by user512 — APPROVED** [review]
Looks good, would you like me to merge without `ARCHITECTURE.md`? I think that might need a bit more time.

Also, I was under the impression by using this library, some payment processors Env var needs to be provided, I think I will have a much better understanding once we have `ARCHITECTURE.md`.

Or is `compensated` just responsible for handling webhook request from payment processor at this moment?

**Comment by zspencer** [comment]
@user512 - Great questions! At present, Compensated _does not_ do any direct interaction with any of the payment processors; it sits quietly listening.

There are use cases where it _could_ do more, and I am all in favor of building them once we have a product in our portfolio (or a client project) that needs it, so we can backport the code.

I think you're right that this is worth merging even without the ARCHITECTURE.md file.  I'm going to remove the WIP and if you wind up merging it before I get to ARCHITECTURE.md I won't complain!

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Developer Documentation for Compensated
+
+Compensated intends to make it safe and easy for people to begin taking payments
+from their customers directly within their application from a variety of payment
+sources. Compensated is packaged and distributed as a Ruby library ([source
```
Comment (user512) [inline docs/README.md:5]: By just reading this statement, I had an impression of this library will handle the full cycle of payment processing where I think an architecture diagram or explicitly saying the current state of this project would be useful.

Comment (zspencer) [reply docs/README.md:5]: Yea, I think we want to move toward that degree of functionality. I wonder if we could ask a few of the folks who have reached out to us what they need the most help with on the "front-end" side; as the current design was optimized for systems where the "front" of the funnel was handled by TypeForm, iOS, and Gumroad respectively.

**Pull request merged**